server/src/main/java/org/apache/druid/client/DirectDruidClient.java

```diff
@@ -203,13 +203,16 @@ public int getNumOpenConnections()
       {
         private final AtomicLong totalByteCount = new AtomicLong(0);
         private final AtomicLong queuedByteCount = new AtomicLong(0);
+        private final AtomicLong channelClosedTime = new AtomicLong(0);
```

Comment:
Maybe `channelClosedTime ` -> `channelSuspendedTime`

Reply:
@QiuMM thanks for pointing this out, will change this. Any idea why we're getting warnings (https://travis-ci.org/apache/incubator-druid/jobs/429402492) for this PR?

Reply:
Not your code cause these fails, don't worry.

Reply:
It seems to be some timeout that didn't cause any error to be printed. I'm not sure what happened but I retried it.

Reply:
@gianm Does the logic for the new metric seem sound?